src/shared/metadataFiles.ts

```diff
@@ -7,8 +7,20 @@ import {
 } from "./metadataInfo";
 import FileUtils from "./fileutils";
 import _ from "lodash";
+import ignore from "ignore";
+import * as fs from "fs";
 
 export default class MetadataFiles {
+  forceignore: any;
+  public constructor() {
+    if (fs.existsSync(".forceignore")) {
```

Comment:
@genoud  As this is on the shared metadataFiles.ts, does this mean diffutils will also utilize ForceIgnore? I have a story AB#61, just checking whether it will solve this as well

Reply:
Yes, we should implement it in diffutils as well.

Reply:
okie Will add a stoy then